src/components/i18n/__tests__/Composition-test.js

```diff
@@ -0,0 +1,270 @@
+import React from 'react';
+
+import Composition from '../Composition';
+
+describe('components/i18n/Composition', () => {
+    test('ComposeString', () => {
```

Comment:
Ideally, test descriptors should explain the expectation rather than the subject under test. The latter is more appropriate for `describe`.